StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -436,14 +436,8 @@ extension PaymentSheet {
                 missingRequirements.formUnion(requirements)
             }
 
-            // - Link Funding Sources contains Bank Account.
-            if elementsSession.linkFundingSources?.contains(.bankAccount) == false {
-                missingRequirements.insert(.linkFundingSourcesMissingBankAccount)
-            }
-
-            // - US Bank Account is *not* an available payment method.
-            if elementsSession.orderedPaymentMethodTypes.contains(.USBankAccount) {
-                missingRequirements.insert(.unexpectedUsBankAccount)
```

Comment:
These checks are part of the computation of `link_supported_payment_methods_onboarding_enabled`.